hardware/static/js/hw_list.js

```diff
@@ -26,7 +26,7 @@ let hw_list = ((hw)=>{
         element.innerHTML=count
     }
     function notifyAvailableItem(item){
-        hw.notify("A "+item.name+" has become available! Click the notification to request.", 
+        hw.notify("A "+item.name+" has become available! Click the notification to request (May not work in your browser).", 
```

Comment:
I think it'd be neater to just remove `Click the notification to request (May not work in your browser).` and make the click target open a page with the item page + button saying "request item"

Reply:
Hey @qaisjp thanks for reviewing. I'm not sure what you mean. The problem is that the notification click event doesn't work on some combinations of browsers/desktops (I found that it works ok on windows both Firefox and Chrome, but it doesn't on a Manjaro with i3). For the cases it works it's pretty convenient, the problem is that I don't have any way to detect if it's going to work for a specific user. Maybe looking in the userAgent..? What would you suggest?